src/plugins/vis_type_timeseries/server/lib/get_fields.ts

```diff
@@ -73,7 +71,13 @@ export async function getFields(
         .toPromise();
     },
   };
-  const { indexPatternString } = await getIndexPatternObject(reqFacade, indexPattern);
+  let indexPatternString = indexPattern;
+
+  if (!indexPatternString) {
+    const index = await reqFacade.getUiSettingsService().get('defaultIndex');
```

Comment:
This will return a default index id, e.x. `ff959d40-b880-11e8-a6d9-e546fe2bba5f`.
To receive an index title, you should get an actual index via `saveObjectsClient` service as in `getIndexPatternObject` helper

You could literally go with 
```
  if (!indexPatternString) {
    const object = await getIndexPatternObject(reqFacade, indexPattern);
    indexPatternString = object.indexPatternString;
  }
```

or you could use the data plugin `indexPatternsService`, where there are `getDefaut` function to retrieve the default index directly. I think this is a better variant.